packages/shared/src/models/IEnvironmentState.ts

```diff
@@ -18,7 +18,7 @@
 
 import type { PodInfo } from '@podman-desktop/api';
 
-export type EnvironmentStatus = 'none' | 'running' | 'starting' | 'stopping' | 'removing' | 'error' | 'unknown';
+export type EnvironmentStatus = 'none' | 'running' | 'starting' | 'error' | 'unknown';
```

Comment:
Why removing the stopping / removing but keeping the starting ? Wouldn't it be better to have instead a `loading` state ? or `transitioning` or else

Reply:
I still need to make some cleanup, I think the 'starting' will also be removed.

This is the state of the pod, I'm not sure yet what to do with it, and it is really necessary. I'll need to continue working on it